### ISSUE TITLE
https://software.broadinstitute.org/software/gaemr/ page not working

### ISSUE THREAD

**Issue by xiaoli0** [broadinstitute/GAEMR#4]
"Your PHP installation appears to be missing the MySQL extension which is required by WordPress."

**Comment by xiaoli0** [comment]
TL;DR 

I need a couple of imgs in the docs. 

![1 1_2](https://github.com/broadinstitute/GAEMR/assets/1276424/dce64567-ed87-474c-ab70-677a6970807a)
![1 1](https://github.com/broadinstitute/GAEMR/assets/1276424/e5566b4e-1d83-42b6-b7ff-92ee0a0a1a44)
![1 3](https://github.com/broadinstitute/GAEMR/assets/1276424/201579cc-f611-4822-a20a-572f63e8297b)
![2 1](https://github.com/broadinstitute/GAEMR/assets/1276424/7919440c-93c6-4bfd-aab5-fde86e71735e)
![2 2](https://github.com/broadinstitute/GAEMR/assets/1276424/c6849534-4c18-4f73-89e1-631518d80e7e)
![2 3](https://github.com/broadinstitute/GAEMR/assets/1276424/9520e721-09cd-443c-a77a-ba0cf92c642f)
![2 4](https://github.com/broadinstitute/GAEMR/assets/1276424/056d8b92-446d-4cd4-97d0-a08b78a7b64d)
![2 5](https://github.com/broadinstitute/GAEMR/assets/1276424/2e0d9de8-f53e-4d2f-bd7c-c98be5df1930)
![2 6](https://github.com/broadinstitute/GAEMR/assets/1276424/d00d7699-2cb4-4b4f-9aa5-c1be6ee76025)
![2 7_2](https://github.com/broadinstitute/GAEMR/assets/1276424/969a2ff3-22bd-42b5-85e8-abc84670dd16)
![2 7](https://github.com/broadinstitute/GAEMR/assets/1276424/c6518e3b-ad47-479e-b0c5-6f9ed97e4342)
![2 8](https://github.com/broadinstitute/GAEMR/assets/1276424/df25ab53-5c22-4a61-8ba6-1fbf24a79ee5)
![2 9](https://github.com/broadinstitute/GAEMR/assets/1276424/e0fe8ebc-519d-4338-9bcc-18e88315edd7)
![2 10](https://github.com/broadinstitute/GAEMR/assets/1276424/f4968260-dfdc-4025-a49b-ab306ed9f773)
![2 11](https://github.com/broadinstitute/GAEMR/assets/1276424/1d63a92e-67b0-4c00-acc4-053c90774049)
![3 1](https://github.com/broadinstitute/GAEMR/assets/1276424/83ca7e1a-5cbc-4a51-a7d1-a1ed825a23e5)
![3 2](https://github.com/broadinstitute/GAEMR/assets/1276424/d24f9c07-2344-46e0-85a4-5c072108bb39)
![4 1](https://github.com/broadinstitute/GAEMR/assets/1276424/cf50fda8-109f-4963-879d-8011d6522af0)
![4 2](https://github.com/broadinstitute/GAEMR/assets/1276424/09b3c46f-cd9a-4b44-a50a-227de7ce3790)
![4 3](https://github.com/broadinstitute/GAEMR/assets/1276424/52611b12-e9c3-4252-8eba-e9a9136567eb)
![4 4](https://github.com/broadinstitute/GAEMR/assets/1276424/ea68ef72-9152-4e92-8afe-6458171d4697)
![4 5](https://github.com/broadinstitute/GAEMR/assets/1276424/58d43875-9c3b-435f-9d06-925ed9889111)
![4 6](https://github.com/broadinstitute/GAEMR/assets/1276424/ca94340d-78a9-443d-b21d-ada9d4ab01d3)
![4 7](https://github.com/broadinstitute/GAEMR/assets/1276424/347b9430-2bef-47bf-9e61-41e3cf9b1b30)
![4 8](https://github.com/broadinstitute/GAEMR/assets/1276424/91cb7103-cb8d-4779-80fd-885f88721697)
![5 1](https://github.com/broadinstitute/GAEMR/assets/1276424/0e599061-3397-4948-a03f-09563be07bbc)
![5 2](https://github.com/broadinstitute/GAEMR/assets/1276424/ee072530-5c96-4029-9dea-2ba3e3e3f3f9)
![5 3](https://github.com/broadinstitute/GAEMR/assets/1276424/1dd67c79-0e2b-4829-a0b3-37aeb3f5649d)
![5 4](https://github.com/broadinstitute/GAEMR/assets/1276424/bc96164a-00df-4257-bea2-13be9f0460f7)
![5 5](https://github.com/broadinstitute/GAEMR/assets/1276424/6a9777ff-388e-48c6-995a-501afc6a5055)
![5 6](https://github.com/broadinstitute/GAEMR/assets/1276424/23bff0ee-61d7-4780-85cc-e4d46c9c7ac3)
![5 7](https://github.com/broadinstitute/GAEMR/assets/1276424/0a3abe29-1178-4ec2-9b54-fbd8904a7bb6)
![5 8](https://github.com/broadinstitute/GAEMR/assets/1276424/1054426d-b673-4144-a521-34294d8d49d3)
![5 9 2](https://github.com/broadinstitute/GAEMR/assets/1276424/ed43cbc1-fc85-4861-b80e-31644b20bbbe)
![5 9](https://github.com/broadinstitute/GAEMR/assets/1276424/fe144bc2-e51e-43ef-a624-966d83e00d47)
![5 10](https://github.com/broadinstitute/GAEMR/assets/1276424/28684707-3995-4b38-bfbb-51918be75114)
![5 11](https://github.com/broadinstitute/GAEMR/assets/1276424/bd975c08-ca9d-48ee-a18a-d004330c13ab)
![5 12](https://github.com/broadinstitute/GAEMR/assets/1276424/d05cb1fd-7a0c-44ce-8088-f9a350f7e189)
![5 13](https://github.com/broadinstitute/GAEMR/assets/1276424/1679a04f-2aed-4cad-b209-323e18a592a3)
![5 14](https://github.com/broadinstitute/GAEMR/assets/1276424/23a187f1-7c14-4809-acee-c1c2c774f663)
![5 15](https://github.com/broadinstitute/GAEMR/assets/1276424/37fa8378-48bc-4ef5-a48d-83626a369d53)
![6 1](https://github.com/broadinstitute/GAEMR/assets/1276424/3ce044b3-6426-4c49-8c41-ec1ac353fae9)
![6 2](https://github.com/broadinstitute/GAEMR/assets/1276424/b6613112-7a9e-45f1-9ac6-493f2644fe1d)
![6 3](https://github.com/broadinstitute/GAEMR/assets/1276424/e054876d-94cc-45ed-9956-fab38fd8d9f8)
![7 1](https://github.com/broadinstitute/GAEMR/assets/1276424/eab88880-2108-4347-8933-c0b2949d6dd9)
![8 1](https://github.com/broadinstitute/GAEMR/assets/1276424/5c6526a1-e598-4511-86a2-427d59b48027)
![8 2](https://github.com/broadinstitute/GAEMR/assets/1276424/bb5e58a1-98e0-46e4-8906-56e54ef5f934)
